### PR TITLE
Compressed site (minified) and added development folder for uncompressed file versions. Closes #121

### LINKED ISSUE #121: Slow page speed
After lots of commits, the page loading speed is very very slow now.
screencapture-developers-google-com-speed-pagespeed-insights-1450630390280
screencapture-developers-google-com-speed-pagespeed-insights-1450630395744

### PR DESCRIPTION
**Pull request by abishekvashok**
Compressed site (minified) and added development folder for uncompressed
code as a part of GCI. Link to task : https://codein.withgoogle.com/dashboard/task-instances/5073449349283840/?sp-page=1

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://Abhi2424shek.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -457,10 +457,4 @@
   github: antoniomdk
   image: antoniomolner.jpg
   twitter: antonio_molner
-  bio: Spanish. 17. Game Developer at Wapmania, Unity, UE4 programmer. Learning HTML5 and C++
-
```
Comment (SaptakS) [inline _data/students.yml:461]: Why have you removed information here?

**Comment by abishekvashok** [comment]
I never modified that file! @SaptakS  The images are compressed now take a look at these facts

**Comment by abishekvashok** [comment]
untitled-1
untitled-2
Sorry ^ the links to facts are as above you can see the increase right and the images where already compressed on another issue

**Comment by abishekvashok** [comment]
Since it has conflicts I will add a new PR with compressed images soon @SaptakS

**Pull request closed without merging**